client/modules/home.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { Card, Typography, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { useSelector, useDispatch } from 'react-redux';
+import { SomeComponent, ProgressCardContent } from 'client/components';
+import { AppStoreState } from 'client/store/reducers';
+import { chapterActions } from 'client/store/actions';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 15,
+  },
+}));
+
+const Home: React.FC = () => {
+  const classes = useStyles();
+
+  const { error, loading, name, description } = useSelector(
```

Comment:
I didn't get it. Wrapping it into a function which would just return what `useSelector` is doing, is just extra overhead, don't you think?

You are suggesting something like this, right:
```jsx
  const chapterSelector = () => useSelector(
    (state: AppStoreState) => ({
      error: state.chapter.error,
      loading: state.chapter.loading,
      name: state.chapter.name,
      description: state.chapter.description,
    }),
  );
```

Reply:
Sorry for the late replay, didn't see this message.
No my idea was to have a
```
const chapterSelector = (state: AppStoreState) => ({...});
const state = useSelector(chapterSelector);
```
since this will probably be used more then once?

Reply:
I am still a bit unsure about this. Can we extract it later. Since later we can find some very strong points of doing this, if this case comes up. If we find duplicate state-maps we can replace this with your idea.

For now I am just thinking it as a re-usable Connected Component, where selectors are not supposed to be duplicated, but the Component could.

Hope you are fine with me here. 😊

Reply:
I was hoping for this to get merged, so that I can open a PR with the TODO list I have mentioned in the PR description. There are some gotcha's with that TODO list, which I want to resolve with everybody's help here.

Reply:
Yeah, I think that's completely fine for now, we can change it later.